__tests__/extension-requests/extension-requests.test.js

```diff
@@ -33,7 +33,7 @@ describe('Tests the Extension Requests Screen', () => {
 
   beforeAll(async () => {
     browser = await puppeteer.launch({
-      headless: 'new',
+      headless: false,
```

Comment:
In my opinion headless false is only for debugging purposes,
https://pptr.dev/guides/debugging/#debugging-methods-for-all-situations

Reply:
Done